frontend/frontend-react/src/components/template/Main.jsx

```diff
@@ -1,20 +1,70 @@
 import './Main.css'
-import React from 'react'
+import React, { Component } from 'react'
 
-export default props =>
-    <main>
-        <div className="content">
-            <div className="post mt-4">
-                <div className="up">
-                    <div className="icon"></div>
-                    <div className="name ml-3">MATHEUS</div>
-                </div>
-                <div className="image"></div>
-                <div className="buttons">
-                    <div className="like"></div>
-                    <div className="comment"></div>
-                    <div className="share"></div>
+const items = [
+    {
+        name: "Quêmia",
+        photo: "https://i1.wp.com/www.toppapeldeparede.com.br/wp-content/uploads/2021/04/Cute-Kimetsu-no-Yaiba-Wallpapers-for.png?fit=996%2C1600&ssl=1",
+        photoName: "Naruto"
+    },
+    {
+        name: "Quêmia",
+        photo: "https://i.pinimg.com/originals/8e/bc/45/8ebc456519f2df51cece20f89d5856c7.jpg",
+        photoName: "Naruto"
+    },
+    {
+        name: "Quêmia",
+        photo: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTyaOfMjrUR-MIBf2nTv_qBWZrTt6rtw5jr_w&usqp=CAU",
+        photoName: "Naruto"
+    },
+    {
+        name: "Quêmia",
+        photo: "https://wallpaperwaifu.com/wp-content/uploads/2019/09/kochou-shinobu-kimetsu-no-yaiba-thumb-1500x844.jpg",
+        photoName: "Naruto"
+    }
+]
+
+export default class Main extends Component {
+    state = { posts: items }
+
+    handle = (e) => {
+        e.target.classList.toggle("red");
+    } 
```

Comment:
This function does nothing and also we have better ways to change/add/remove classNames from React. You can use ternary operators:

```jsx
<button className={`default-class ${isTrue ? 'true-class' : 'not-true-class'}`}>
```

Where `isTrue` is any variable.